Update the OD comparison table, and add links and references to `Beatmap/Overall difficulty` (#8144)
ppy/osu-wiki: wiki/Beatmap/Overall_difficulty/en.md

| 100 | `140 - 8 * OD` |
| 50 | `200 - 10 * OD` |

> #### Table comparison

Reviewer: seems unnecessary, table can go in same section as above

Reviewer 2: i don't have too much of an opinion, but if it's removed then the same should be done in the ar article

Author: okay

> ![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers")

Reviewer: the OD values for DT/HT shouldn't be kept without explanation -- or maybe at all

the numbers for OD there are saying "a map with this OD on nomod would have the same hit window for 300s", but it would be different for 100 and 50, so it's kind of misleading.

Author: "but it would be different for 100 and 50" -- what do you mean here? could you coin a full explanation, perhaps?

Reviewer: given an OD you can find hit windows using the formulas above, but given the time-scaled hit windows from DT or HT, you can't map them back to a single OD value

for example look at OD5+DT
300 window is 33.33ms = nomod OD7.78 (this is shown on the chart)
100 window is 66.67ms = nomod OD9.17
50 window is 100ms = nomod OD10

when i made the chart, I put the equivalent nomod OD for the 300 hit window, since that's probably what most players care to see. but without explanation it can be misleading

Author: thanks, that makes it clear. does this work? https://github.com/ppy/osu-wiki/pull/8144/commits/38a8843588ca147b28f12cfd3c3c04e53521f28b

| 100 | `140 - 8 * OD` |
| 50 | `200 - 10 * OD` |

> ![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, referenced OD values are only based on 300's hit windows, and would be different for 100 and 50.")

Reviewer 2: ```suggestion
![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, the referenced OD values are only valid for the hit windows of 300s, and would be different for 100s and 50s.")
```

> ![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, the referenced OD values are only valid for the hit windows of 300s, and would be different for 100s and 50s.")

Reviewer: I think saying "referenced OD values" is wrong because those are the final result and not referenced by anything on the chart. maybe somethin like "the OD values are based on..." or "the OD values are calculated using..."

Author: this is not the actual OD value. instead, it's a value used as an example, the hit window of which for a 300 is the same as the hit window for a given OD and a combination of mods. doesn't that make it a value used as a reference? perhaps I should have used `reference` instead of `referenced`? 

if I am still wrong, which I admit I may be, well, fine, I'll make an edit (or you can too, whichever is faster and more comfortable)

Reviewer 2: no the word itself just doesn't make sense at all here. there's nothing that we're referencing from.

```suggestion
![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, the shown OD values are only valid for the hit windows of 300s, and would be different for 100s and 50s.")
```

Author: okay, done

## Sliders and spinners

> In [osu!](/wiki/Game_mode/osu!), [sliders](/wiki/Gameplay/Hit_object/Slider) will reward a 300 as long as they are hit within the 50's hit window. This is sometimes referred to as *slideracc* and is removed in [ScoreV2](/wiki/Game_modifier/ScoreV2)

Reviewer 3: I know this is in the original, but I believe the whole sentence about "slideracc" is back to front (it was *added* in ScoreV2 / lazer, not removed). Also I think "slideracc" should be replaced with "slider accuracy" as there are very few references to the weird abbreviation on google.

Reviewer 2: this was probably meant to be "slider leniency" instead, but it can go both ways

## osu!catch

Overall difficulty is visible when viewing beatmap information, but it does not affect gameplay.

## References

> [^judgement-rounding-ref]: [Discord message by spaceman_atlas (2022-05-06) in #osu-wiki](https://discord.com/channels/188630481301012481/218677502141399041/972241866382798889)

Reviewer 2: ```suggestion
[^judgement-rounding-ref]: [Discord message by spaceman_atlas (2022-05-06) in #osu-wiki in osu!dev](https://discord.com/channels/188630481301012481/218677502141399041/972241866382798889)
```
